### PR TITLE
Fixes for Travis tests - AR 5.2.x

### PR DESCRIPTION
**Pull request by eanders**
Charlie, let me know if you'd prefer the schema updates separated from the test fixes.  The tests won't pass without those two changes, and I'm not sure how necessary the schema changes are without the tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@ language: ruby
 sudo: false
 bundler_args: --without "db2 oracle sqlserver"
 
+services:
```
Comment (Slike9) [inline .travis.yml:5]: The file already has `services` section bellow..

File: test/test_create.rb [hunk 2]
```diff
@@ -50,9 +50,8 @@ def test_create_no_id
 
   def test_create_generated_keys
     # Not all databases support columns with multiple identity fields
-    if defined?(ActiveRecord::ConnectionAdapters::PostgreSQL) ||
-       defined?(ActiveRecord::ConnectionAdapters::SQLite3)
-
+    postgres = defined?(ActiveRecord::ConnectionAdapters::PostgreSQL) && Suburb.connection.class == ActiveRecord::ConnectionAdapters::PostgreSQL
+    if postgres
```
Comment (Slike9) [inline test/test_create.rb:54]: What about using `skip`?

File: test/fixtures/db_definitions/mysql.sql [hunk 3]
```diff
@@ -52,7 +52,7 @@ create table product_tariffs (
 );
 
 create table suburbs (
-    city_id int not null,
+    city_id int not null auto_increment,
```
Comment (Slike9) [inline test/fixtures/db_definitions/mysql.sql:55]: Suspect we don't need this change as we are skipping the test.

**Comment by eanders** [comment]
@Slike9 Thanks for the feedback.  I've adjusted and it appears all tests are still passing.

File: .travis.yml [hunk 4]
```diff
@@ -12,14 +12,15 @@ script:
   - "rake mysql:test"
 
 rvm:
-  - 2.4.10
+  - 2.4.9
   - 2.5.8
-  - 2.6.6
-  - 2.7.1
+  - 2.6.5
+  - 2.7.0
```
Comment (codeodor) [inline .travis.yml:18]: Why are these being decremented?

File: test/fixtures/db_definitions/mysql.sql (hunk 3 above)
Comment (eanders) [reply test/fixtures/db_definitions/mysql.sql]: This is used in other tests which will fail without it.

File: .travis.yml (hunk 4 above)
Comment (eanders) [reply .travis.yml]: At the time of writing, travis was reporting not having access to the newer rubies.  That seems to have been fixed.  I've pushed an updated travis.yml.

**Comment by cfis** [comment]
Yes, let's split this up. One patch for Travis. As for the schema changes, I'm not understanding the point of the Suburb check?

Going to close this pull request - let's separate these out.

**Pull request closed without merging**